warden/group/manager_sql.go

```diff
@@ -123,6 +125,28 @@ func (m *SQLManager) RemoveGroupMembers(group string, subjects []string) error {
 	return nil
 }
 
+func (m *SQLManager) ListGroups(limit, offset int64) ([]string, error) {
+	if limit < 0 {
```

Comment:
I think it should be fine to set limit to 0 (or 1) if it's below 0

Reply:
OK